packages/contracts-bedrock/test/universal/BenchmarkTest.t.sol

```diff
@@ -59,43 +87,99 @@ contract GasBenchMark_L1StandardBridge_Deposit is CommonTest {
     }
 
     function test_depositETH_benchmark_0() external {
+        // Skip if the test is running in coverage.
+        if (vm.isContext(VmSafe.ForgeContext.Coverage)) {
+            vm.skip(true);
+        }
+
+        // Setup
         vm.pauseGasMetering();
+        uint256 startingGas = gasleft();
         setPrevBaseFee(vm, address(optimismPortal2), 1 gwei);
         vm.resumeGasMetering();
+
+        // Test
         l1StandardBridge.depositETH{ value: 500 }(50000, hex"");
+
+        // Assert
+        vm.pauseGasMetering();
+        assert(startingGas - gasleft() < 500_000);
+        vm.resumeGasMetering();
     }
 
     function test_depositETH_benchmark_1() external {
+        // Skip if the test is running in coverage.
+        if (vm.isContext(VmSafe.ForgeContext.Coverage)) {
+            vm.skip(true);
+        }
+
+        // Setup
         vm.pauseGasMetering();
+        uint256 startingGas = gasleft();
         setPrevBaseFee(vm, address(optimismPortal2), 10 gwei);
         vm.resumeGasMetering();
+
+        // Test
         l1StandardBridge.depositETH{ value: 500 }(50000, hex"");
+
+        // Assert
+        vm.pauseGasMetering();
```

Comment:
We should be able to delete all the manual pause/resumeGasMetering + gasleft() calls and just call `vm.lastCallGas()` after the call being tested

Reply:
Will try this